exonum-java-binding-time-service/src/main/java/com/exonum/binding/time/serialization/UtcZonedDateTimeSerializer.java

```diff
@@ -15,29 +15,32 @@
  *
  */
 
-package com.exonum.binding.common.serialization;
+package com.exonum.binding.time.serialization;
 
-import static com.exonum.binding.common.serialization.SerializationUtils.checkLength;
 import static com.google.common.base.Preconditions.checkArgument;
 
+import com.exonum.binding.common.serialization.Serializer;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
 import java.time.Instant;
 import java.time.ZoneOffset;
 import java.time.ZonedDateTime;
 
-enum UtcZonedDateTimeSerializer implements Serializer<ZonedDateTime> {
+/**
+ * ZonedDateTime serializer. Only serializes values with UTC zones, throws an exception otherwise.
+ */
+public enum UtcZonedDateTimeSerializer implements Serializer<ZonedDateTime> {
   INSTANCE;
 
-  private static final int SERIALIZED_ZDT_SIZE = Long.BYTES + Integer.BYTES;
+  private static final int SERIALIZED_DATE_TIME_SIZE = Long.BYTES + Integer.BYTES;
 
   @Override
   public byte[] toBytes(ZonedDateTime value) {
-    checkArgument(value.getZone() == ZoneOffset.UTC,
-            "Serialized ZonedDateTime value should be in UTC");
+    checkArgument(value.getZone() == ZoneOffset.UTC, "ZonedDateTime value should be in UTC, but was %s",
+        value.getZone().toString());
```

Comment:
Will `value.getZone()` work (IIRC, checkX invoke `toString` for you and lazily)?